hazelcast/src/main/java/com/hazelcast/internal/config/AbstractConfigLocator.java

```diff
@@ -113,6 +115,11 @@ public boolean isConfigPresent() {
      */
     public abstract boolean locateDefault();
 
+    public static boolean isDefaultConfigFile(File file) {
```

Comment:
I think this is not a good criterion for checking if `file` is a default config file.
Assuming I have a file named `/opt/hazelcast/hazelcast-default.xml` and use it to start hazelcast with `-Dhazelcast.config=/opt/hazelcast/hazelcast-default.xml`, should the above scenario fail? I assume it's loaded from the "default" file if it was loaded by `AbstractConfigLocator#loadDefaultConfigurationFromClasspath`, not if just the filename matches.

Reply:
https://github.com/hazelcast/hazelcast/pull/20446/commits/87e10baddea47859440058d7dc87a55d75d00195